doc/ref_model/chapters/appendix-a.md

```diff
@@ -67,6 +70,35 @@ Without arguing for or against SR-IOV, CNTT provides the following anecdotes whi
 **CPU Instruction Sets and Extensions:**
 <content needed>
 
+**Raw Disk Device Mapping**
+<content needed>
+
+Overview of technology:
+
+It is possible to map SAN LUNs or local disks to a virtual unit (VM or container), giving direct hardware access to the
+specified device.  As this is a form of PCI passthrough, it should be considered as a violation of the abstration
+principle.
+
+See: vfio-pci for qemu.
+
+**Remote Direct Memory Access**
+<content needed>
+
+RDMA allows direct memory access from the memory of one computer to the memory of another computer without involving the
+operating system or CPU. The transfer of memory is offloaded to the RDMA-capable Host Channel Adapters (HCA).  As this
+requires specific binary code (driver) to be present this should be considered a violation of the abstraction principle.
+
+Ref: https://github.com/qemu/qemu/blob/master/docs/rdma.txt
+
+**Persistent Memory**
+<content needed>
+
+Persistent Memory, also known as Non-Volatile Memory (NVM), is capable of maintaining data even after a power outage and is
+typically implemented as virtual NVDIMM.  As this is often backed by a host physical file, this technology must be examined to
+determine if it violates the abstraction principle.
+
+Ref: https://docs.pmem.io/persistent-memory/getting-started-guide/creating-development-environments/virtualization/qemu
```

Comment:
@markbeierl Please provide a rationale for (v)PMem. This one is not as clear to me, b/c if you're proposing to expose the "memory" as some kind of block device or especially as a SCSI device, you still effectively have a "disk", which is obviously something that is already available, and a filesystem is one of the easiest/most expedient ways to manage a big block of bytes. This is even more especially the case, if the virtual persistent memory is, in fact, implemented via a real disk drive at the physical layer (even if it's local NVMe) .  It seems really getting our money out of this paradigm (persistent memory statically mapped into a directly addressable CPU memory region), would require a highly specialized workload (at least at this time), where the kernel loader is aware the contents of this memory region is "already loaded" (i.e. it's not a file system or disk), so it doesn't try to load said contents after every kernel restart, which more akin to an embedded application, than a cloud. Also, our general direction is to persist as little as possible (except where we don't :-). Not saying it couldn't happen, and times do change, but I'm not sure we're there yet. Looking forward to seeing the rationale and use cases . Thanks, and interesting concept! -Mark

Reply:
Current Intel implementation (DIMM capacities of 128, 256 and 512GB) has two modes:

1. non-persistent large memory beyond DRAM,
2. Persistent where app needs to access the memory and be enabled so node after reboot it can find the data (instead of reloading TBs from somewhere).

It can be used for in-memory DBs like store state for decomposed functions, or CDN Live Streaming high-res channels.